.storybook/static/healthcare-theme.css

```diff
@@ -62,7 +62,7 @@
 --color-secondary-dark: #107b0d;
 --color-secondary-darker: #0d600a;
 --color-secondary-darkest: #094507;
---color-secondary-light: #0d600a;
+--color-secondary-light: #59ac56;
```

Comment:
@pwolfert I think these came from your last PR but I'm not sure why they're showing up here. Maybe this change was just missed in a push?

Reply:
Nevermind I see you put in a PR for this :)